Mathlib/Data/ZMod/Basic.lean

```diff
@@ -748,6 +752,17 @@ theorem val_coe_unit_coprime {n : ℕ} (u : (ZMod n)ˣ) : Nat.Coprime (u : ZMod
   rw [Units.val_mul, val_mul, nat_cast_mod]
 #align zmod.val_coe_unit_coprime ZMod.val_coe_unit_coprime
 
+lemma isUnit_iff_coprime (m n : ℕ) : IsUnit (m : ZMod n) ↔ m.Coprime n := by
+  refine ⟨fun H ↦ ?_, fun H ↦ (unitOfCoprime m H).isUnit⟩
```

Comment:
I've heard contradicting opinions on `=>` versus `↦`, but in this file it looks like `=>` takes the majority

Reply:
This is certainly an artifact of the way mathport is formatting the code.